test/dnf5-plugins/CMakeLists.txt

```diff
@@ -1 +1 @@
-add_subdirectory(copr_plugin)
+#add_subdirectory(copr_plugin)
```

Comment:
We should add a comment describing why the copr_plugin test is disabled.

Reply:
Okay, I'll add it.